grails-app/controllers/org/pih/warehouse/core/StockMovementItemsRequestCommand.groovy

```diff
@@ -0,0 +1,20 @@
+package org.pih.warehouse.core
+
+import grails.validation.Validateable
+import org.pih.warehouse.api.PaginationCommand
+
+class StockMovementItemsRequestCommand extends PaginationCommand implements Validateable {
+
+    String id
+
+    Integer stepNumber
+
+    Boolean createNew = true
+
+    static constraints = {
+        id nullable: false
```

Comment:
I don't mind doing it explicitly, but `nullable: false` is the default setting, so we could get rid of the `id` here.

Reply:
+ you've added the `Validateable`, and those constraints, but I can't see you eventually validating the command.
validating is a wrong word, because it's validated while being bound, but I can't see you eventually returning the errors.
We could skip it for now, since previously there was not such check for the params, but just wanted to emphasize, that currently the errors would be "swallowed".